keps/sig-network/563-dual-stack/README.md

```diff
@@ -971,10 +980,13 @@ CoreDNS will need to make changes in order to support the plural form of
 endpoint addresses. Some other considerations of CoreDNS support for
 dual-stack:
 
-- Because service IPs will remain single-family, pods will continue to access
-  the CoreDNS server via a single service IP. In other words, the nameserver
-  entries in a pod's /etc/resolv.conf will typically be a single IPv4 or single
-  IPv6 address, depending upon the IP family of the cluster's service CIDR.
+- For dual-stack services, CoreDNS read the endpoint slices generated for each IPFamily
+  and create the corresponding A and AAAA records for the service.
+- For single-stack services, CoreDNS will operate as is. Service IPs will remain 
+  single-family, pods will continue to access the CoreDNS server via a single service 
+  IP. In other words, the nameserver entries in a pod's /etc/resolv.conf will typically
+  be a single IPv4 or single IPv6 address, depending upon the IP family of the cluster's 
```

Comment:
should we make a statement about resolv.conf `nameserver` when the DNS service ITSELF is dual-stack?